client/verta/docs/_templates/class.rst

```diff
@@ -5,6 +5,10 @@
 .. autoclass:: {{ objname }}
     :members:
     :inherited-members:
+    {% if objname.endswith("Enum") -%}
+    :undoc-members:
+    :member-order: bysource
+    {% endif -%}
```

Comment:
`Enum` variants are technically class members, and there isn't a clean way to attach meaningful docstrings to them, so we have to clumsily tell our documentation, "If a class's name ends with `Enum`, render its undocumented members".

Reply:
This bit gives you the `OPTION = 'option'` style listing of all the possible options at the end of the doc string I assume?

Reply:
Yes